packages/wallet-connectors/src/BrowserWallet.ts

```diff
@@ -73,6 +73,10 @@ export class BrowserWalletConnector implements WalletConnector, WalletConnection
         return this;
     }
 
+    async ping() {
+        return undefined;
+    }
```

Comment:
```suggestion
    async ping() {}
```
You don't need to return undefined, unless it's to avoid some linting. :shrimp:

Reply:
It is indeed to appease the linter.